src/server.c

```diff
@@ -67,16 +67,16 @@ int send_response(int fd, char *header, char *content_type, void *body, int cont
     time(&uet);
     time_info = localtime(&uet);
 
-    sprintf(response, "%s\n"
-                      "Date: %s"
-                      "Connection: close\n"
-                      "Content-Length: %i\n"
-                      "Content-Type: %s\n"
-                      "\n"
-                      "%s",
-            header, asctime(time_info), content_length, content_type, body_content);
-
-    int response_length = strlen(response);
+    int response_length =
+        sprintf(response,
+                "%s\n"
+                "Date: %s"
+                "Connection: close\n"
+                "Content-Length: %i\n"
+                "Content-Type: %s\n"
+                "\n"
+                "%s",
+                header, asctime(time_info), content_length, content_type, body_content);
```

Comment:
I mentioned this today but I think having it in writing will help reference later: this will handle text files fine, but it won't work for jpgs or pngs. You'll have to figure out another way to copy the body into the response if you want to do that.

Reply:
Oversight in the spec on my part. I thought the image handling was part of the stretch. I’ll get memcpy() added in tomorrow.